tests/integration/components/frost-fixed-table-test.js

```diff
@@ -327,4 +344,154 @@ describe(test.label, function () {
       })
     })
   })
+
+  describe('events', function () {
+    let dispatcher
+
+    beforeEach(function () {
+      dispatcher = sandbox.stub()
+      this.setProperties({
+        actions: {dispatcher},
+        fixedColumns: fixedColumnsWithCustomRenderers
+      })
+
+      this.render(hbs`
+        {{frost-fixed-table
+          callbackHandler=(action 'dispatcher')
+          columns=fixedColumns
+          hook=myHook
+          items=heroes
+        }}
+      `)
+
+      return wait()
+    })
+
+    describe('the header', function () {
+      const headerRow = -1  // differentiates from data row 0
+
+      describe('the left section', function () {
+        describe('when an event is triggered by a renderer', function () {
+          beforeEach(function () {
+            // FIXME: Fix this to use qualifiers on '...renderer-input' hook
+            // the renderer has a hook, but the input inside of it doesn't have the right qualifiers.
+            const cellRenderer = $hook('myTable-header-left-cell-renderer', {column: 0})
+            $('input', cellRenderer).val('foo').trigger('input')
```

Comment:
If you want to avoid the global `$` (or `Ember.$` you can use `.find()` on the previously selected thing. Also, I'd put a `$` in front of the variable name if it's a `jQuery` object:

```js
const $celRenderer = $hook(...)
$cellRenderer.find('input').val('foo').trigger('input')
```
```